src/client/themes/light/index.ts

```diff
@@ -58,24 +104,23 @@ const lightTheme: DefaultTheme = {
       },
     },
 
-    walletButton: {
-      background: '#000000',
-      color: '#FFFFFF',
-    },
+    // walletButton: {
+    //   background: '#000000',
+    //   color: '#FFFFFF',
+    // },
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6